docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/LightGbm.cs

```diff
@@ -17,7 +17,7 @@ public static void Example()
             // Create in-memory examples as C# native class.
             var examples = DatasetUtils.GenerateRandomMulticlassClassificationExamples(1000);
 
-            // Convert native C# class to IDataView, a consumble format to ML.NET functions.
+            // Convert native C# class to IDataView, a consumable format to ML.NET functions.
```

Comment:
>native C# class [](start = 23, length = 15)

please spell out the full type: List<Something>

Reply:
I added more specificity, without spelling out the full type. 

If we want full-types, we should spell them out instead of using `var`.  Because the compiler will make sure it's always correct and catch the changes. Having it in the comments will go stale in no time.

So far we've been using `var` in the samples, so I'll keep it for consistency.

---
In reply to: [261050690](https://github.com/dotnet/machinelearning/pull/2771#discussion_r261050690) [](ancestors = 261050690)